test/powershell/Language/Scripting/Debugging/DebuggerCommand.Tests.ps1

```diff
@@ -214,14 +214,13 @@ Describe 'Basic debugger command tests' -tag 'CI' {
         }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Basic debugger command tests.Callstack (k, Get-PSCallStack) command should show the current call stack.Should only have CallStackFrame output from the callstack command
```powershell
Expected the value to have type [System.Management.Automation.CallStackFrame] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Debugging\DebuggerCommand.Tests.ps1: line 217
217:             $result['k'].Output | Should -BeOfType System.Management.Automation.CallStackFrame
```